lib/ehmdmc_client.rb

```diff
@@ -4,16 +4,35 @@ module EHMDMCClient
 
   def self.validate?(hmdmc)
     r = connection.get('?hmdmc=' + sanitise(hmdmc))
```

Comment:
While testing it I notice that when the hmdmc service is down, connection.get raises Faraday::ConnectionFailed; the AJAX call returns a 500 error but the content is a rails stacktrace instead of an error json, so the UI does not display any message in this case.  Should we add it as a bug?

Reply:
That sounds like some kind of bug.